src/store/epics/transfer.ts

```diff
@@ -90,7 +92,31 @@ export const amountCalculations$: RootEpic = (
     withLatestFrom(minBalance$),
     map(([remainingBalance, minBalance]) => remainingBalance < minBalance),
   )
-  const updateInsufficientFunds$ = insufficientFunds$.pipe(map(insufficientFunds))
+  const updateInsufficientFunds$ = insufficientFunds$.pipe(
+    withLatestFrom(state$),
+    map(
+      ([
+        isInsufficientFunds,
+        {
+          connections: { currentCurrency },
+          transfer: {
+            targetPayeeIsExisted,
+            formData: { amount, targetPayee },
+          },
+        },
+      ]) => {
+        if (isValidPublicKey(targetPayee) && !targetPayeeIsExisted && Number(amount) < 0.02) {
```

Comment:
I think here instead of hard coding 0.02, we should be using the same logic for determining the minimum amount left in the wallet. For example, where we calculate (2 x Base Reserve)